src/adapters/MeetingsJSONAdapter.js

```diff
@@ -501,6 +506,20 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
     }));
   }
 
+  /**
+   * Switches the speaker control.
+   *
+   * @param {string} ID  Meeting ID
+   * @param {string} speakerID  ID of the speaker device to switch to
```

Comment:
Choose:)
- Id of the speaker device to switch to
- Device id of the speaker to switch to